plugin/scripts/package_pip_packages.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+LIBS_DIR="plugin/teksi_wastewater/libs"
```

Comment:
what about git submodule?

qgis-plugin-ci easily package a submodule, that would simplify the packaging part.

Reply:
git attributes allow to configure what we don't want in the archive, that's a good thing for packaging at least